pytplot/QtPlotter/CustomAxis/DateAxis.py

```diff
@@ -1,38 +1,78 @@
 import pyqtgraph as pg
 import time
+import numpy as np
```

Comment:
Just realized this was left there after I removed a test on the code. Will remove it now.